docs/token-guides/wsteth-bridging-guide.md

```diff
@@ -0,0 +1,353 @@
+
+In short, to deploy upgradable custom (still native) bridges on L1 and L2 and an upgradable token on L2, all governed by the Lido DAO on L1 ([Aragon Agent contract](https://etherscan.io/address/0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c)) via a dedicated [AAVE-like governance executor](https://github.com/lidofinance/governance-crosschain-bridges/) contract on L2. This architecture is proposed to provide the following capabilities.
+
+1. Have the ability to pass additional data and revamp logic. Specifically for the bridging rebasable stETH (requires passing wstETH/stETH rate). The stETH token is not a general-purpose token but an asset built on top of the liquid staking protocol, which has moving and upgradable parts.
+2. Let the token be "future proof" and avoid the potential need to do high-cost liquidity migration as Ethereum continues to evolve and new standards like ERC-2612/ERC-1271 get added.
+3. Have the ability to pause/resume token bridging in case of an emergency or upgrade operation planning.
+
+## The Lido DAO bridging endpoints recognition
+
+Lido DAO can recognize the bridged wstETH endpoints by means of a signaling snapshot. For example, it has happened for: [Base](https://snapshot.org/#/lido-snapshot.eth/proposal/0x8b35f64fffe67f67d4aeb2de2f3351404c54cd75a08277c035fa77065b6792f4), [zkSync Era](https://snapshot.org/#/lido-snapshot.eth/proposal/0xe35f56a6117599eeb9dbef982613c0545710d91403a828d1fba00ab21d5188f3), [Mantle](https://snapshot.org/#/lido-snapshot.eth/proposal/0x1d38c11b27590ab5c69ca21c5d2545d53b7f5150dada7e05f89d500ede5becad), [Linea](https://snapshot.org/#/lido-snapshot.eth/proposal/0x9382624eeee68a175dd7d1438347dbad4899ba0d2bfcf7c3955f087cb9f5cfc4).
+
+If Lido DAO recognizes the bridged wstETH endpoints, in general, it means:
+- Lido DAO has levers to upgrade the dedicated token bridge and token contracts;
+- the integration would be highlighted on the frontend pages: landing, widget, and ecosystem pages; however,  there are no in-house bridging UIs yet;
+- contracts are under the Lido's bug bounty program;
+- there is a monitoring and alerting suite built on Forta;
+- marketing campaign: blog, tweets, incentives for the liquidity bootstrapping long-term, potential reward share program cases.
+
+Usually, the Lido DAO recognizes the bridged wstETH endpoints if the specific set of security and design recommendations are met. These recommendations are set out in [On-bridging-solution](#On-bridging-solution) section in paragraphs **R1-R8**. The rest of the recommendations (**R9-...**) are also important and foster the recognition's likelihood.
+
+If only recommendations [R1-R4](https://www.hypernative.io/) are met the token might get acknowledged by the NEW as meeting security and future-proof baseline and/or get possible [LoL](https://research.lido.fi/t/liquidity-observation-lab-lol-liquidity-strategy-and-application-to-curve-steth-eth-pool/5335) incentives at max (still if safu but using the LoLs own criteria).
```

Comment:
Consider rephrasing, e.g.,

Only if the recommendations R1-R4 are observed, does the token have a chance of being acknowledged by NEW as meeting the security and future-proofing baseline. At most, the token may receive LoL incentives if the appropriate criteria are met.